src/status_im/ui/screens/browser/views.cljs

```diff
@@ -109,6 +117,8 @@
     [react/view
      [icons/icon :icons/arrow-right]]]
    [react/view {:flex 1}]
+   (when dapp?
```

Comment:
we already have "open chat" button here, can we move share to options, same as in chat? @errorists

Reply:
Sure please do so, we can also fit 5 icons inside the toolbar so if there's space for one more left, share can be the last one.

Reply:
@alwx what do you think, can we move it in options?

Reply:
yes, will do!

Reply:
@flexsurfer @errorists where should I place this "Options" button? Should it be instead of "Wallet" button or to the right of it? Or maybe it needs to be on the bottom?

Reply:
@errorists wasn't it intended to keep `Share chat` beneath `...`? 

<img src="https://user-images.githubusercontent.com/15470435/45743277-f10ac380-bbfb-11e8-92c3-46f952d3d3d8.png" width=250/>

Reply:
My bad @rachelhamlin it was about browser screen, not chat .. :sweat_smile:

Reply:
I'd suggest to to move both the wallet and share into the overflow ••• menu, this way we'll have a consistent UX between chat and dapps. There's already an issue opened for the Wallet icon in the navbar here https://github.com/status-im/status-react/issues/5802

Reply:
Aha, I see. Agree with @errorists.